src/adapters/analytics/ga.js

```diff
@@ -56,7 +55,8 @@ exports.enableAnalytics = function ({ provider, options }) {
       sendBidResponseToGa(bid);
 
     } else if (eventObj.eventType === BID_TIMEOUT) {
-      _timedOutBidders = args.bidderCode;
+      const bidderArray = args;
```

Comment:
I think the `const` is nice for readability/making it obvious what's being passed, would you prefer without?